conf/config-default.yaml

```diff
@@ -169,9 +169,30 @@ nginx_config:                     # config for render the template to generate n
 
   http:
     enable_access_log: true        # enable access log or not, default true
-    access_log: "logs/access.log"
-    access_log_format: "$remote_addr - $remote_user [$time_local] $http_host \"$request\" $status $body_bytes_sent $request_time \"$http_referer\" \"$http_user_agent\" $upstream_addr $upstream_status $upstream_response_time \"$upstream_scheme://$upstream_host$upstream_uri\""
-    access_log_format_escape: default       # allows setting json or default characters escaping in variables
+    # access_log_format: "$remote_addr - $remote_user [$time_local] $http_host \"$request\" $status $body_bytes_sent $request_time \"$http_referer\" \"$http_user_agent\" $upstream_addr $upstream_status $upstream_response_time \"$upstream_scheme://$upstream_host$upstream_uri\""
+    # access_log: "logs/access.log"
+    # access_log_format_escape: default       # allows setting json or default characters escaping in variables
+    access_log:
```

Comment:
We can't accept a break change, the new change should be compatible with the old style, like:
https://github.com/apache/apisix/blob/2707fda3eb7b23d995ac23ecf4a17000dd765855/apisix/cli/ops.lua#L410